msgpack-core/src/main/java/org/msgpack/core/ExtensionTypeHeader.java

```diff
@@ -3,16 +3,21 @@
 import static org.msgpack.core.Preconditions.*;
 
 /**
- * Header of the extended types
+ * Header of the Extension types
  */
-public class ExtendedTypeHeader {
+public class ExtensionTypeHeader {
     private final byte type;
     private final int length;
 
-    ExtendedTypeHeader(byte type, int length) {
-        checkArgument(length >= 0, String.format("length must be >= 0: %,d", length));
-        this.length = length;
+    public ExtensionTypeHeader(byte type, int length) {
```

Comment:
Why don't you add java doc here to suggest that users can use checkedCastToByte?